core/templates/components/state-editor/state-editor-properties-services/state-param-changes.service.spec.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit test for the State Param Changes service.
+ */
+
+import { TestBed } from '@angular/core/testing';
+/* eslint-disable max-len */
+import { StateParamChangesService } from 'components/state-editor/state-editor-properties-services/state-param-changes.service.ts';
```

Comment:
Is there no way to do this without violating the eslint rules?

Reply:
I have tried different permutations and combinations but didn't work.

Reply:
Can't you just move the file name to next line and maybe split the string with a '+'?

Reply:
@aks681 I tried that but it's not working.

Reply:
What about moving 'components...' to the next line?

Reply:
@kevinlee12 still it violets the rules of max length.